lint.sh

```diff
@@ -17,7 +17,7 @@ install(){
     virtualenv $venv
   fi
   . $venv/bin/activate
-  pip install -c constraints.txt -r test-requirements.txt >/dev/null
+  pip install -c constraints.txt -r test-requirements.txt -r requirements.txt >/dev/null
```

Comment:
Hey @hughsaunders , why is this change necessary all of a sudden?

Reply:
I think its usage of jmespath, I can't remember where it is though.

Reply:
So what is the distinction between requirements and test-requirements?

Reply:
Deferring to @hughsaunders on this one, since he made that specific change.

Reply:
Note this same change is in https://github.com/rcbops/rpc-gating/pull/629/ ... so I don't think this is needed by this specific PR?

Reply:
Why is it here then? Looking at the lint jobs, they are not failing.